staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5877,7 +5877,11 @@ type NodeRuntimeHandlerFeatures struct {
 	// +featureGate=RecursiveReadOnlyMounts
 	// +optional
 	RecursiveReadOnlyMounts *bool `json:"recursiveReadOnlyMounts,omitempty" protobuf:"varint,1,opt,name=recursiveReadOnlyMounts"`
-	// Reserved: UserNamespaces *bool (varint 2, for consistency with CRI API)
+	// UserNamespaces is set to true if the runtime handler supports UserNamespaces.
```

Comment:
"... if the runtime handler supports UserNamespaces, including idmapped mounting".

But do we want this to be SO specific?  Or is more possible to phrase more generically, like "...if the runtime handler supports UserNamespaces, including PersistentVolumes"

Reply:
I think the latter sounds good to me. Let me go ahead and update the description.